### PR TITLE
Allow member to hold multiple posts in a body

### LINKED ISSUE #244: body member cannot occupy both a membership post and a leadership post
Relates to https://github.com/popolo-project/popolo-spec/issues/125

I need to keep track of the posts that make a member part of a body and any leadership posts they have in the body (like "speaker"). This now seems impossible with pupa's duplicate checking.

```python
pupa.exceptions.DuplicateItemError: attempt to import data that would conflict with data already in the import: 
{'role': '1st Vice Chair', 'post_id': None, 'on_behalf_of_id': None, 
  'start_date': '2014-07-01', 'person_id': 'ocd-person/c976773c-c13e-485c-bbd6-0a007970f029', 
  'label': '', 'end_date': '2015-06-30', 
  'organization_id': 'ocd-organization/42e23f04-de78-436a-bec5-ab240c1b977c', 'extras': {}} 
(already imported as Mark Ridley-Thomas in Board of Directors (Board Member))
```

Do we want to change how we do do duplicate checking? Attn @jamesturk @jpmckinney

### PR DESCRIPTION
**Pull request by fgregg**
closes #244

### CODE REVIEW

**Comment by fgregg** [comment]
This is ready for review @jamesturk @jpmckinney

File: pupa/scrape/jurisdiction.py [hunk 1]
```diff
@@ -42,9 +42,6 @@ def as_dict(self):
                 'legislative_sessions': self.legislative_sessions,
                 'feature_flags': self.feature_flags, 'extras': self.extras, }
 
-    def get_session_list(self):
```
Comment (jamesturk) [inline pupa/scrape/jurisdiction.py]: is this change accidental?

Comment (fgregg) [inline pupa/scrape/jurisdiction.py]: No, it should have been part of https://github.com/opencivicdata/pupa/pull/243. I can put into a separate PR, if you want.

**Comment by jpmckinney** [comment]
Looks good to me - not sure if there's a consequence to removing `get_session_list`.

**Comment by fgregg** [comment]
bump, @jamesturk

**Comment by jamesturk** [comment]
haven't had a chance to really consider what we'd lose by removing get
session list, but I'm good with the actual changes here

**Comment by fgregg** [comment]
Okay, I'll split this out.

**Comment by fgregg** [comment]
I've removed the changes to `get_session_list` so this should be ready to go.

**Pull request merged**